docs/lint_errors.md

```diff
@@ -227,14 +195,11 @@ The `README.md` files for a project are very important and must meet some requir
 
 ## Error #7 - Pipeline and container version numbers ## {#7}
 
-> This test only runs when `--release` is set or `$TRAVIS_BRANCH` is equal to `master`
+> This test only runs when `--release` is set
 
-These tests look at `process.container` and `$TRAVIS_TAG`, only
-if they are set.
+These tests look at `process.container` only if they are set.
```

Comment:
As above, we don't want to lose this functionality. Here it's checking for the specific release version tag that was set and making sure that the code corresponds to it. Again will need to be replaced with `GITHUB_REF` I think.

Reply:
https://github.com/nf-core/tools/blob/25335b387589008838c83fb0a73808cc0f9a7cbb/nf_core/lint.py#L732-L734

Reply:
See https://github.com/nf-core/tools/pull/502/commits/d7c06b61e7abfef8298c63bef0d9817e72949bb3